lib/components/viewers/stop-viewer.js

```diff
@@ -71,7 +71,7 @@ class StopViewer extends Component {
 
   componentDidMount () {
     // Load the viewed stop in the store when the Stop Viewer first mounts
-    this.props.findStop({ stopId: this.props.viewedStop.stopId })
+    this.props.fetchStopInfo(this.props.viewedStop)
```

Comment:
This `fetchStopInfo` could be made into an action without an argument that handles retrieving the viewedStop prop from the redux state.

Reply:
I don't think we're gaining much with that kind of change. I'd prefer to have it accept an argument .